### DISCUSSION TITLE
Appointment options - Scrollbar

### DISCUSSION THREAD

**Discussion by markusthiel** [lukevella/rallly#1257] [Ideas]
When many appointment options are available, a scrollbar and two small arrows appear to scroll horizontally through the appointment options. Unfortunately, this is really hard to see. I know myself that there are more options, but everyone I send these appointments to only votes on the ones that are immediately visible and doesn't scroll further through the timeline. This should be made more visible so that it is self-explanatory.

**Comment by lukevella** [comment]
Hey @markusthiel, the list is actually scrollable with a scrollbar already. On some OSes the scrollbar might be hidden until you hover over it or trigger the scrolling. Or perhaps you're self-hosting an older version?
